wp-job-manager.php

```diff
@@ -410,9 +410,11 @@ public function frontend_scripts() {
 				$select2_args['dir'] = 'rtl';
 			}
 
+			$select2_args['width'] = '100%';
+
 			wp_localize_script(
-				'select2', 'job_manager_select2_multiselect_args',
-				apply_filters( 'job_manager_select2_multiselect_args', $select2_args )
```

Comment:
Do we need to deprecate this? Or has it not been released yet?

Reply:
It hasn't been released yet. All `select2` stuff is in this release.